### PR TITLE
Bunch of small fixes related to the minimiser.

### PR DESCRIPTION
**Pull request by thehrh**

### CODE REVIEW

File: pisa/resources/settings/minimizer_settings/bfgs_settings_fac1e7_eps1e-4_mi50.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "options":{
     "value": {
       "disp"   : 0,
-      "ftol"   : 2e-5,
```
Comment (sboeser) [inline pisa/resources/settings/minimizer_settings/bfgs_settings_fac1e7_eps1e-4_mi50.json]: Should this change not be reflected in the file name? Or is the filename correct and the value "wrong"?

File: pisa/analysis/llr/LLHAnalysis.py [hunk 2]
```diff
@@ -184,7 +186,7 @@ def find_opt_scipy(fmap, template_maker, params, minim_settings,
         alt_dict_flags = {}
         alt_dict_flags['warnflag'] = alt_minim_result.status
         alt_dict_flags['task'] = alt_minim_result.message
-        if minim_result.has_key(jac):
+        if minim_result.has_key('jac'):
```
Comment (sboeser) [inline pisa/analysis/llr/LLHAnalysis.py]: Why did this not throw an error before?

File: pisa/resources/settings/minimizer_settings/bfgs_settings_fac1e7_eps1e-4_mi50.json (hunk 1 above)
Comment (thehrh) [inline pisa/resources/settings/minimizer_settings/bfgs_settings_fac1e7_eps1e-4_mi50.json]: The value now reflects the filename.

File: pisa/analysis/llr/LLHAnalysis.py (hunk 2 above)
Comment (thehrh) [inline pisa/analysis/llr/LLHAnalysis.py]: It did.

**Pull request merged**